PWGDQ/Core/MCProng.cxx

```diff
@@ -100,13 +103,22 @@ void MCProng::SetUseANDonSourceBits(int generation, bool option /*=true*/)
   fUseANDonSourceBitMap[generation] = option;
 }
 
+//________________________________________________________________________________________________________________
+void MCProng::SetSignalInTime(int generation, bool intime /*=false*/)
```

Comment:
The "generation" parameter is not needed since the "intime" holds for the whole prong history